bin/rad-replicate

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env radicle
+
+(load! (find-module-file! "prelude.rad"))
+(load! (find-module-file! "monadic/project.rad"))
+
+(import prelude/strings '[reverse-string] :unqualified)
+
+(def help
+  "rad-replicate - Replicate radicle projects.
+
+  This can be used to make your machines more available (e.g., readable when
+  you are offline, or writable and readable if you are behind a firewall).
+
+   Usage:
+        rad-replicate [URL]
+
+  If not provided, the URL defaults to http://replicate.radicle.xyz
+  ")
+
+(def default-url
+  "Default replication URL"
+  "http://replicate.radicle.xyz")
```

Comment:
You could use `default-url` in the help text, then there is less updating if this changes.